USER-INTERFACE-INTERFACE.md

```diff
@@ -372,7 +391,7 @@ database password. If you want to know whether the password you have is the corr
 * `earningWalletAddressOpt`: The wallet address for the earning wallet. This is not secret, so
   if you don't get this field, it's because it hasn't been set yet.
 
-* `gasPrice`: The Node will not pay more than this number of wei for gas to complete a transaction.
+* `gasPrice`: The Node will not pay more than this number of Gwei for gas to complete a transaction.
```

Comment:
@kauri-hero you may want to review the changes in this file to make sure it's consistent with other documentation and how these same concepts are explained to users and GUI developers and testers.